setup.cfg

```diff
@@ -40,6 +40,7 @@ install_requires =
   psutil
   qdarkstyle
   setuptools
+  sqlalchemy
```

Comment:
That's a heavy dependency and it does the same thing as Peewee does. I understand that it's the quickest way to get a permanent job store for APScheduler, but not too happy.

Reply:
Sure, that's true. Looking at the [implementation of the SQLAlchemy jobstore](https://github.com/agronholm/apscheduler/blob/master/apscheduler/jobstores/sqlalchemy.py), it seems doable to port it to either peewee or plain sqlite. However, I'm not too keen right now to be honest.